gatsby-node.esm.js

```diff
@@ -74,6 +74,91 @@ exports.createPages = async ({ actions, graphql }) => {
       },
     })
   })
+  const v4CauseSpecificHomepages = await graphql(`
+    {
+      allCauseSpecificJsonDataJson(limit: 1000) {
+        edges {
+          node {
+            data {
+              sections {
+                Mission {
+                  subtitle
+                  text
+                  title
+                }
+                TFACIntro {
+                  img1
+                  img1Subtext
+                  img2
+                  img2Subtext
+                  img3
+                  img3Subtext
+                  title
+                }
+                charityIntro {
+                  introImg1
+                  introImg1Subtext
+                  introImg2
+                  introImg2Subtext
+                  subTitle
+                  title
+                }
+                landing {
+                  subtitle
+                  title
+                  ctaImg {
+                    childImageSharp {
+                      gatsbyImageData(width: 200, formats: AUTO)
```

Comment:
we'll want to move away from formats auto and add more things here.  Can do this in the query builder tool